cost/alibaba/Alibaba_Multiple_Files/Alibaba_Daily.pt

```diff
@@ -0,0 +1,426 @@
+long_description "This is to be used when daily granularity (multiple files) from Alibaba. Used when costs have been exported on a daily basis for consumption to OSS."
+severity "low"
+category "Cost"
+default_frequency "daily"
+info(
+  version: "0.1",
+  provider: "Alibaba",
+  service: "Common Bill Ingestion",
+  policy_set: "Common Bill Ingestion"
+)
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_billing_period" do
+  type "string"
+  label "Month To Ingest"
+  description "Month to process bills for."
+  default "Current Month"
+  allowed_values "Current Month", "Specific Month", "Previous Month"
+end
+
+parameter "param_specific_period" do
+  type "string"
+  label "Billing Period"
+  description "Billing period to process bills for in YYYY-MM format. Only relevant if Specific Month is selected for Month To Ingest."
+  default "2023-11"
+  allowed_pattern /^(19|20)\d\d-(0[1-9]|1[0-2])$/
+end
+
+parameter "param_cbi_endpoint" do
+  type "string"
+  label "Flexera CBI Endpoint"
+  description "Name of the Flexera CBI endpoint to use. Ex: cbi-oi-alibaba-alibabacloud"
+  default "cbi-oi-alibaba-alibabacloud"
+end
+
+parameter "param_account_id" do
+  type "string"
+  label "Alibaba Cloud Account ID"
+  description "Account ID for Alibaba Cloud Account"
+end
+
+parameter "param_region" do
+  type "string"
+  label "Alibaba Cloud Region"
+  description "Region of the Alibaba Cloud Object Storage bucket containing the cost and usage reports. Ex: oss-cn-shanghai"
+end
+
+parameter "param_bucket" do
+  type "string"
+  label "Alibaba Cloud Billing Report Bucket"
+  description "Alibaba Cloud Object Storage bucket containing the Billing reports. EX: flexera-billing-data"
+end
+
+###############################################################################
+# Authentication
+###############################################################################
+
+credentials "auth_aws" do
+  schemes "aws"
+  label "Alibaba"
+  description "Select the Alibaba Cloud Credential from the list."
+  tags "provider=aws"
+end
+
+credentials "auth_flexera" do
+  schemes "oauth2"
+  label "flexera"
+  description "Select FlexeraOne OAuth2 credentials"
+  tags "provider=flexera"
+end
+
+###############################################################################
+# Pagination
+###############################################################################
+
+###############################################################################
+# Datasources
+###############################################################################
+
+#CALCULATE BILLING PERIOD FOR RETRIEVING ALIBABA CLOUD BILLING DATA
+datasource "ds_billing_period" do
+  run_script $js_billing_period, $param_billing_period, $param_specific_period
+end
+
```

Comment:
Accompanying 'script' should follow the datasource - e.g., in this case, this datasource should be followed by the script `js_billing_period` block currently on line 294.

See https://github.com/flexera-public/policy_templates/blob/master/cost/azure/azure_china_cbi/azure_china_cbi.pt or https://github.com/flexera-public/policy_templates/blob/master/cost/oracle/oracle_cbi/oracle_cbi.pt for examples